libs/electron/src/window.main.ts

```diff
@@ -13,6 +13,7 @@ const WindowEventHandlingDelay = 100;
 export class WindowMain {
   win: BrowserWindow;
   isQuitting = false;
+  isClosing = false;
 
   private windowStateChangeTimer: NodeJS.Timer;
   private windowStates: { [key: string]: any } = {};
```

Comment:
Using `WindowState` instead of any would give us types

Reply:
May I ask you what do you mean in this context?

Reply:
I meant using the type `WindowState` instead of `any`. This will give us type-information for anywhere where we access `windowStates`.

```suggestion
  private windowStates: { [key: string]: WindowState } = {};
```